router/routes.js

```diff
@@ -0,0 +1,312 @@
+const router = require('express').Router();
+const dotenv = require('dotenv');
+const bcrypt = require('bcrypt');
+const sgMail = require('@sendgrid/mail');
+const passport = require('passport');
+const jwt = require('jsonwebtoken');
+const verify = require('../middleware/verify');
+const CLient = require('../model/client');
+const Address = require('../model/address');
+dotenv.config();
+sgMail.setApiKey(process.env.API_KEY);
+// route to register user
+router.post('/register', async (req, res) => {
+  try {
+    if (req.body.password != req.body.confirm_password) {
+      return res.send('Password do not match');
+    }
+    const salt = await bcrypt.genSalt(10);
+    const password = await bcrypt.hash(req.body.password, salt);
+    const user = await CLient.create({
+      first_name: req.body.first_name,
+      last_name: req.body.last_name,
+      username: req.body.username,
+      email: req.body.email,
+      password: password,
+    });
+    const msg = {
+      to: req.body.email,
+      from: process.env.SENDER_EMAIL,
+      subject: 'User Registered',
+      text: `Your email have been registerd with username ${req.body.username}`,
+      html: `<strong>Your email have been registerd with username ${req.body.username}</strong>`,
+    };
+    await sgMail.send(msg).then(() => {
+      console.log('email sent');
+    });
+    res.status(200).send({
+      status: '1',
+      message: 'Success',
+      data: {
+        message: `user created with username : ${req.body.username} and email : ${req.body.email}`,
+      },
```

Comment:
send `message` only one time not two times (no need to send message in data too, you can use data field/key just to send the details like user or address etc)
also **status key** is sufficient to identify that api is resolved successfully or got an error